### PR TITLE
`explore`: consolidate padding config, handle ByteStream, tweak naming+comments

### PR DESCRIPTION
**Pull request by rgwood**
Some minor changes to `explore`, continuing on my mission to simplify the command in preparation for a larger UX overhaul:

1. Consolidate padding configuration. I don't think we need separate config points for the (optional) index column and regular data columns in the normal pager, they can share padding configuration. Likewise, in the binary viewer all 3 columns (index, data, ASCII) had their left+right padding configured independently.
2. Update `explore` so we use the binary viewer for the new `ByteStream` type. `cat foo.txt | into binary | explore` was not using the binary viewer after the `ByteStream` changes.
3. Tweak the naming of a few helper functions, add a comment

I've put the changes in separate commits to make them easier to review.

### CODE REVIEW

File: crates/nu-explore/src/views/binary/binary_widget.rs [hunk 1]
```diff
@@ -12,6 +12,9 @@ use crate::{
     views::util::{nu_style_to_tui, text_style_to_tui_style},
 };
 
+// Padding between segments in the hex view
```
Comment (sholderbach) [inline crates/nu-explore/src/views/binary/binary_widget.rs:15]:
```suggestion
/// Padding between segments in the hex view
```

**Review by sholderbach — COMMENTED** [review]
Awesome, thanks for simplifying this. Having a tighter configuration with a few obvious knobs should make this more usable.

Haven't run it yet (so can't say much about the `ByteStream` stuff, cc @devyn , @IanManske )

**Comment by devyn** [comment]
Seems to work, but I can't use page up and page down. I'm using:

```nushell
open ~/.cargo/bin/nu | into binary | explore
```

**Comment by rgwood** [comment]
@devyn I think that's a preexisting issue, I can't use page up/down in the binary viewer on an older version of Nushell either. I think there's a lot of stuff that isn't quite finished in the binary viewer :(

**Pull request merged**